scripts/lake/link-locale-lake.sh

```diff
@@ -8,9 +8,10 @@ rm -rf clients/banking/node_modules/.vite
 rm -rf clients/onboarding/node_modules/.vite
 
 # create js module to export lake path
-echo 'import path from "node:path";
+echo 'const path = require("pathe");
+
 // This file is generated by scripts/lake/link-local-lake.sh
 // You can edit it in case your lake repo is not in the same directory as this repo
-export default {
-  lake: path.resolve(process.cwd(), "..", "lake"),
+module.exports = {
+  lake: path.resolve(__dirname, "../lake"),
```

Comment:
are you sure this is working? because in my memories `__dirname` refers to module path (so in this case `/scripts/lake`) whereas `process.cwd()` refers to the path where the command is run (in our case with `yarn dev`, it's `/`).  
But maybe I'm completely wrong and should go back learning how nodejs works for path

Reply:
@Epimodev `__dirname` refers to the actual file path (so here, repository root)